app/assets/stylesheets/donate.scss

```diff
@@ -0,0 +1,23 @@
+.block-text {
+  @include span-columns(6);
+  @include shift(3);
+}
+
+.button {
+  @include span-columns(3);
+  @include shift(5.25);
+  padding-bottom: 20px;
+}
+
+ul {
+  @include shift(0.5);
+  list-style-type: circle;
+}
+
+button > a {
+  color: #fff;
+
+  &:hover {
+    color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.